src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+# Metrics
+
+This section describes how to enable metrics reporting on the Policy Server.
+
+We are going to use [Prometheus](https://prometheus.io/) to scrape metrics exposed by the Policy
+Server.
+
+## Install Prometheus
```

Comment:
Maybe we should define a new 2nd level section called `Prerequisites` with the following 3rd level sections:

  1. Install OpenTelemetry
  2. Install Prometheus
  3. Install Kubewarden

Reply:
This is already the case, if I'm not misunderstanding your comment.

The index looks like:

7. Operator Manual
  7.1. OpenTelemetry: intructions on setting up a Kubernetes cluster + opentelemetry
  7.2.  Metrics: setting up jaeger + kubewarden with jaeger integration
  7.3. Tracing: setting up prometheus + kubewarden with prometheus integration

Or do you refer to something else?

Reply:
Oh true, the numbering only appears on the rendered book. I'm happy with that.

Reply:
I see. I would still add some text at the top of this file to ensure people are aware of the requirements that have been covered by the other sections of the book.

To put it in a more direct way: ensure you went through "OpenTelemetry" section before doing anything written here